packages/node-resolve/src/index.js

```diff
@@ -201,7 +201,7 @@ export function nodeResolve(opts = {}) {
 
       const importeeIsBuiltin = builtins.has(importee);
 
-      if (importeeIsBuiltin && (!preferBuiltins || !isPreferBuiltinsSet)) {
+      if (importeeIsBuiltin) {
         // The `resolve` library will not resolve packages with the same
```

Comment:
Wouldn't the logic here become superfluous by using the new option in the updated `resolve` package?

Reply:
Yep! Am looking into that in a different branch though.